docs/user-manuals/jobsidecarterminator.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Job Sidecar Terminator
+---
+
+**FEATURE STATE:** Kruise v1.4.0
+
+In Kubernetes world, we hope that Pods controlled by job-type workload enter `Completed` phase immediately once the task executed in main container is completed.
+However, in the scene where the these Pods have long-running sidecar containers, Pod may be out of our expectations because the long-running sidecar container cannot be terminated when the main containers completed.
+
+To solve such problem, we proposal a controller, namely sidecar terminator, to watch and terminate sidecar containers for such job-type Pods when its main containers completed.
+
+## Requirements
+
+ - Enabled `JobSidecarTerminator` feature gate when installing/upgrading Kruise (defaults to `disabled`).
+ - Enabled `KruiseDaemon` feature gate when installing/upgrading Kruise (defaults to `enabled`).
+
+## Usage
+
+It is very easily to use this feature, you just need to add a special env to the sidecar containers you want to terminate in the Pods:
+```yaml
+kind: Job
+spec:
+  template:
+    spec:
+      containers:
+        - name: sidecar
+          env:
+            - name: KRUISE_TERMINATE_SIDECAR_WHEN_JOB_EXIT
+              value: "true"
+        - name: main
+... ...
+```
+
+### Notes
+
+- Your sidecar container must respond the `SIGTERM` signal, and the `PID 1` should `exit 0` when received this signal.
```

Comment:
Your sidecar container must respond the SIGTERM signal, and the entrypoint should exit 0 when received this signal.